web/gui/bundle_dashboard.py

```diff
@@ -101,5 +111,15 @@ def write_makefile():
     BASEPATH.joinpath('Makefile.am').write_text(MAKEFILEDATA)
 
 
+def list_changed_files():
+    '''Create a list of changed files, and set it in an environment variable.'''
```

Comment:
I take it that the list of files is and will remain small enough so that we can stay below the system's environment size limit.

Reply:
For the moment, it seems exceedingly unlikely that we will ever have a list of file names that takes up more than 128 kiB given that the list is currently at approximately 5.2 kiB.